packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -88,18 +86,14 @@ contract Bridge is EssentialContract, IBridge {
         if (message.user == address(0)) revert B_INVALID_USER();
 
         // Check if the destination chain is enabled.
-        (bool destChainEnabled, address destBridge) =
-            isDestChainEnabled(message.destChainId);
+        (bool destChainEnabled,) = isDestChainEnabled(message.destChainId);
 
         // Verify destination chain and to address.
         if (!destChainEnabled) revert B_INVALID_CHAINID();
         if (message.destChainId == block.chainid) {
             revert B_INVALID_CHAINID();
         }
 
-        if (message.to == address(0)) revert B_INVALID_TO();
```

Comment:
remove checking for the `to` address, they are considered normal cases and are handled properly by the Bridge on the destination chain.